tests/Task/AbstractTaskTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\SchedulerBundle\Task;
+
+use PHPUnit\Framework\TestCase;
+use SchedulerBundle\Task\AbstractTask;
+
+/**
+ * @author Jérémy Vancoillie <contact@jeremyvancoillie.fr>
+ */
+final class AbstractTaskTest extends TestCase
+{
+    public function testTaskPrioritySetter(): void
+    {
+        $abstractTask = $this->getMockForAbstractClass(AbstractTask::class, ['name' => 'foo']);
+
+        self::assertEquals(1000, $abstractTask->setPriority(1000)->getPriority());
+        self::assertEquals(1000, $abstractTask->setPriority(1001)->getPriority());
+        self::assertEquals(-1000, $abstractTask->setPriority(-1000)->getPriority());
+        self::assertEquals(-1000, $abstractTask->setPriority(-1001)->getPriority());
+        self::assertEquals(5, $abstractTask->setPriority(5)->getPriority());
+    }
```

Comment:
Consider adding these tests inside the `NullTaskTest` class, I'm not a big fan of using `getMockForAbstractClass` and `AbstractTask` is never instantiated so there's no benefice in my opinion to add a test class for it.

Reply:
ok i understand, i didn't want the `NullTask` to interfere with any options. I wanted to make sure to test the master method. i will change that